src/app/app.config.ts

```diff
@@ -90,6 +90,12 @@ export interface EditionConfig {
     proseVersesToggler: boolean;
     defaultTextFlow: TextFlow;
     verseNumberPrinter: number;
+    annotatorColors: AnnotatorColors;
+}
+
+export interface AnnotatorColors {
+    note: string;
+    highlight: string[];
```

Comment:
Since it is an array using the plural as property name is better. Change to `highlights`